tests/integration/cartography/intel/aws/test_s3.py

```diff
@@ -153,7 +153,9 @@ def test_load_s3_policies(neo4j_session, *args):
         )
         for n in nodes
     ]
-    assert actual_nodes == expected_nodes
+    assert len(actual_nodes) == 3
```

Comment:
While you're in here, it might be a good opportunity to refactor it to use `check_rels()` and `check_nodes()`, but this works too.